services/kafka/client.go

```diff
@@ -51,27 +50,26 @@ var (
 
 // NewClient creates a new Kafka client
 func NewClient(clientOptions clientoptions.TerraformClientOptions, fileConfig *fileconfiguration.FileConfig) *Client {
-	config := kafka.NewConfiguration(clientOptions.Credentials.Username, clientOptions.Credentials.Password, clientOptions.Credentials.Token, clientOptions.Endpoint)
+	config := shared.NewConfiguration(clientOptions.Credentials.Username, clientOptions.Credentials.Password, clientOptions.Credentials.Token, clientOptions.Endpoint)
 
-	if os.Getenv(constant.IonosDebug) != "" {
-		config.Debug = true
-	}
 	config.MaxRetries = constant.MaxRetries
 	config.MaxWaitTime = constant.MaxWaitTime
-	config.HTTPClient = http.DefaultClient
-	config.HTTPClient.Transport = shared.CreateTransport(clientOptions.SkipTLSVerify, clientOptions.Certificate)
 	config.UserAgent = fmt.Sprintf(
 		"terraform-provider/_ionos-cloud-sdk-go-kafka/%s_hashicorp-terraform/%s_terraform-plugin-sdk/%s_os/%s_arch/%s",
 		kafka.Version, clientOptions.TerraformVersion, meta.SDKVersionString(), runtime.GOOS, runtime.GOARCH, //nolint:staticcheck
 	)
-	client := Client{sdkClient: *kafka.NewAPIClient(config),
-		fileConfig: fileConfig,
+
+	client := Client{
```

Comment:
@rmocanu-ionos shouldn't this be:

```
client := Client{
sdkClient: *kafka.NewAPIClient(config),
fileConfig: fileConfig,
}
```
?

Reply:
that is correct, fixed it